DataSet.rst

```diff
@@ -477,17 +477,21 @@ to in some cases refer to other DataSets, they should in no cases refer to a Dat
 +-----------------------+------------------------------------------------+---------------------------+
 | Bam ExternalResource  | PacBio.Index.BamIndex                          | .bam.bai                  |
 +-----------------------+------------------------------------------------+---------------------------+
-| Bam ExternalResource  | PacBio.Index.PacBioIndex                       | .bam.pbi                  |
+|                       | PacBio.Index.PacBioIndex                       | .bam.pbi                  |
 +-----------------------+------------------------------------------------+---------------------------+
 | Fasta ExternalResource| PacBio.Index.SamIndex                          | .fasta.fai                |
 +-----------------------+------------------------------------------------+---------------------------+
-| Fasta ExternalResource| PacBio.Index.SaWriterIndex                     | .fasta.sa                 |
+|                       | PacBio.Index.SaWriterIndex                     | .fasta.sa                 |
 +-----------------------+------------------------------------------------+---------------------------+
-| Fasta ExternalResource| PacBio.Index.Indexer                           | .fasta.index              |
+|                       | PacBio.Index.Indexer                           | .fasta.index              |
```

Comment:
We might want to move `.fasta.index` and `.fasta.contig.index` down the list or even remove them entirely, since they are no longer generated in 4.0.